promenade/templates/generate/validate-cluster.sh

```diff
@@ -0,0 +1,6 @@
+{% include "common_validation.sh" with context %}
+
+EXPECTED_NODE_COUNT={{ config['Cluster']['nodes'] | length }}
+wait_for_ready_nodes $EXPECTED_NODE_COUNT
+
```

Comment:
Would we not also validate all pods as part of the promenade deployment are running as well? Nodes could all be ready, but if this is inclusive of skydns, armada, etc. at this stage (as well as any other foundation element) we would want to ensure those are all running and ready in addition to the nodes themselves?